client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -80,12 +83,29 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
             }
         }
 
+        MapType mapType = mapper.getTypeFactory().constructMapType(HashMap.class, String.class,
+                SystemResourceLimits.class);
+        Map<String, SystemResourceLimits> systemResourceLimits = new HashMap<>();
+        try {
+            Optional<Path> filePath = deTilde(systemLimits);
```

Comment:
so you're only supporting a file input, what about command line?

Reply:
Do we need to support a command line json iput? Doesn't seems user-friendly.

Reply:
Why isn't that user friendly to allow more than one option? Command line is probably necessary for our tests anyway. 

We support both file and cmd for the configuration input.